tests_pytest/pytorch_tests/e2e_tests/test_weights_manual_selection_bitwidth.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2025 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import pytest
+
+import model_compression_toolkit as mct
+import torch
+from torch.nn import Conv2d
+from tests.common_tests.helpers.tpcs_for_tests.v2.tpc import get_tpc
+from model_compression_toolkit.target_platform_capabilities.constants import BIAS, PYTORCH_KERNEL
+from model_compression_toolkit.target_platform_capabilities.constants import KERNEL_ATTR, BIAS_ATTR, WEIGHTS_N_BITS
+from model_compression_toolkit.core.common.network_editors import NodeTypeFilter, NodeNameFilter
+from model_compression_toolkit.core import CoreConfig
+from model_compression_toolkit.target_platform_capabilities.tpc_models.imx500_tpc.latest import get_op_quantization_configs, generate_tpc
+from mct_quantizers import PytorchQuantizationWrapper
+
+kernel_weights_n_bits = 8
+bias_weights_n_bits = 32
+activation_n_bits = 8
+
+def representative_data_gen(shape=(3, 8, 8), num_inputs=1, batch_size=2, num_iter=1):
+    for _ in range(num_iter):
+        yield [torch.randn(batch_size, *shape)] * num_inputs
+
+def get_tpc():
+    base_cfg, _, default_config = get_op_quantization_configs()
```

Comment:
Same as in the other test, here you also shouldn't use the imx500_tpc but rather define a small, dedicated tpc for the test case/cases you want to check